trace-sdk/src/main/java/io/bitrise/trace/data/collector/memory/MemoryDataCollector.java

```diff
@@ -9,6 +9,6 @@
 
     @Override
     public long getIntervalMs() {
-        return 15 * 1000;
```

Comment:
this seemed like redundant maths - i hope it's okay i simplified it :)

Reply:
I usually keep the components separate, to make it more understandable, but in this simple case when we just multiply with 1000 I agree with the simplification. 👍